sentry_sdk/tracing.py

```diff
@@ -155,6 +155,28 @@ def add(self, span):
             self.spans.append(span)
 
 
+class MeasurementValue:
```

Comment:
Thoughts on a typed dict instead of another class? This adds another layer to the serialization unnecessarily

Reply:
done